WalletWasabi.Fluent/ViewModels/AddWallet/SelectWalletCreationOptionViewModel.cs

```diff
@@ -76,7 +76,7 @@ private async Task OnImportWalletAsync(string walletName)
 		private async Task OnCreateWalletAsync(string walletName)
 		{
 			var dialogResult = await NavigateDialogAsync(
-				new CreatePasswordDialogViewModel("Type the password of the wallet and click Continue.", enableEmpty: true, enableCancel: Services.WalletManager.HasWallet()));
+				new CreatePasswordDialogViewModel("Type the password of your wallet. It is required to load the wallet, and cannot be changed later.", enableEmpty: true, enableCancel: Services.WalletManager.HasWallet()));
```

Comment:
```suggestion
				new CreatePasswordDialogViewModel("Type the password of your wallet. It is required to load the wallet, and can not be changed later.", enableEmpty: true, enableCancel: Services.WalletManager.HasWallet()));
```

I'm not even sure if `cannot` is actually correct :smile: 
I don't like `can't` that much, so let's go with two words, maybe.

Reply:
There is no difference. Both are acceptable but one is used more according to the answers here: https://english.stackexchange.com/questions/4510/why-is-cannot-spelled-as-one-word

Basically people have different opinion as with lot of things in English. Let me know if `can not` looks better I can change it.

Reply:
interesting, thanks.

what are we using elsewhere in the codebase? I believe we have mainly `can not`, and maybe no `cannot` at all...

Reply:
Adding more text makes things even worse because people will not read it and larger it is less chances to be read. Do not add more text. We have already added "useful" text before and it doesn't work. This is not different, it is useless and will help no one.
NACK